packages/server/lib/cloud/api.ts

```diff
@@ -166,9 +227,32 @@ export type CreateRunOptions = {
   timeout?: number
 }
 
+let preflightResult = {
+  encrypt: true,
+  apiUrl,
+}
+
+let recordRoutes = apiRoutes
```

Comment:
why set to local var? if easier to reason with, can we rename at the import level so this is consistent ?